core/src/main/scala/libretto/CoreLib.scala

```diff
@@ -1267,10 +1265,10 @@ class CoreLib[DSL <: CoreDSL](val dsl: DSL) { lib =>
         override def getL[B](next: Getter[A, B])(using B: Cosemigroup[B]): (S |+| T) -⚬ (B |*| (S |+| T)) =
           id[S |+| T].bimap(self.getL(next), that.getL(next)) > factorL
 
-        override def extendJunction(using A: Junction.Positive[A]): Junction.Positive[S |+| T] =
+        override def extendJunction(using Junction.Positive[A]): Junction.Positive[S |+| T] =
```

Comment:
I would usually prefer to not name unused `using` parameters. But this principle should applied uniformly.

I would revert this case here in case removing unneeded param names is not liked.

Reply:
👍 I too tend to not name them, but sometimes historically it was used explicitly and after refactoring the explicit use disappeared, but the name stayed.

I don't put too much value on consistency throughout the codebase in this trivial matter, though.